From 8f65fae5cf5d0ef3fb258e0680eac59ade795d66 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Thu, 26 Nov 2015 17:44:25 +0800
Subject: [PATCH 0750/5242] MLK-11910 mxc: mlb: fix irq unsigned_compare issue

commit  7d8c44f3c8a16fe5f4d4e6abfdc916728f9b91bb from
https://source.codeaurora.org/external/imx/linux-imx.git

After coverity code check, it tips:
unsigned_compare: This less-than-zero comparison of an unsigned value is never true

Interrupt variable must be signed type.

Signed-off-by: Fugang Duan <B38611@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/mlb/mxc_mlb.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/mxc/mlb/mxc_mlb.c b/drivers/mxc/mlb/mxc_mlb.c
index fa6cff1..cc899c0 100755
--- a/drivers/mxc/mlb/mxc_mlb.c
+++ b/drivers/mxc/mlb/mxc_mlb.c
@@ -378,9 +378,9 @@ struct mlb_data {
 	void __iomem *membase;	/* mlb module base address */
 	struct gen_pool *iram_pool;
 	u32 iram_size;
-	u32 irq_ahb0;
-	u32 irq_ahb1;
-	u32 irq_mlb;
+	int irq_ahb0;
+	int irq_ahb1;
+	int irq_mlb;
 	u32 quirk_flag;
 };
 
-- 
1.7.9.5

